DataFormats/Detectors/ITSMFT/common/src/ClusterTopology.cxx

```diff
@@ -144,7 +144,7 @@ std::ostream& operator<<(std::ostream& os, const ClusterTopology& topology)
   return os;
 }
 
-void ClusterTopology::getCOGshift(int nRows, int nCols, const unsigned char patt[Cluster::kMaxPatternBytes], int& rowShift, int& colShift)
+void ClusterTopology::getCOG(int nRows, int nCols, const unsigned char patt[Cluster::kMaxPatternBytes], int& rowShift, int& colShift, float& xCOG, float& zCOG)
```

Comment:
This function computes the COG position within the bounding box. In ``BuildDictionary.cxx`` (#L32) a similar piece of code is used, even if the position is expressed in terms of the pixel pitch. It could be a good idea to make this function a (static) method of ``ClusterPattern`` or ``ClusterTopology``.

Reply:
Well, here we are not building the dictionary. So calling methods the BuildDictionary class does not see to be logical to me.
But indeed, BuildDictionary.cxx contains quite a similar piece of code.  I will see how this code duplication could be minimised.

Reply:
I meant exactly to use a unique function both in BuildDictionary and here